cross/tvheadend/patches/002-fix-recording-dir.patch

```diff
@@ -0,0 +1,12 @@
+# This ensures that the default recording directory is not placed in package system directories, independent of the DSM version
+--- src/dvr/dvr_config.c.orig	2018-02-27 18:58:45.957196424 +0100
++++ src/dvr/dvr_config.c	2018-02-27 19:00:17.210138337 +0100
+@@ -282,7 +282,7 @@
+ 
+   /* Try to figure out a good place to put them videos */
+ 
+-  homedir = getenv("HOME");
++  homedir = "/usr/local/tvheadend/var";
```

Comment:
I have a preference for `/var/packages/tvheadend/target/var`

Reply:
@ymartin59 `/usr/local/tvheadend` is used consistently for recording directories, not only here but also during the upgrade procedures when moving files from package root into `/usr/local/tvheadend/var`. If we change it here, we should change the upgrade scripts too and that would require a whole new round of testing of the upgrade scripts, which I would very much like to defer (I'm getting tired). I can look at it again for the next update, but if it is just about preference, I'd very much like to leave it as is for now.